CHANGELOG.md

```diff
@@ -4,24 +4,62 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
-## [v1.0.0](https://github.com/voxpupuli/puppet-k8s/tree/v1.0.0) (2023-04-27)
+## [v1.1.0](https://github.com/voxpupuli/puppet-k8s/tree/v1.1.0) (2024-02-21)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-k8s/compare/v1.0.0...v1.1.0)
+
+**Implemented enhancements:**
+
+- Make cni\_plugins download url configurable with a template string [\#82](https://github.com/voxpupuli/puppet-k8s/pull/82) ([olavst-spk](https://github.com/olavst-spk))
```

Comment:
Oh so this looks like a breaking change because it drops a parameter in a class that isn't private? In that case we have to do a 2.0.0 release.

Reply:
hmm and i think i overlooked something in this pr, will open an issue.

Reply:
I guess #83 is also a breaking change then?

Reply:
Added labels and changed metadata to 2.0.0